app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'cocktail',
+  'lib/auth-errors',
+  'lib/promise',
+  'stache!templates/partial/coppa-age-input',
+  'views/form',
+  'views/mixins/floating-placeholder-mixin'
+], function (Cocktail, AuthErrors, p, Template, FormView, FloatingPlaceholderMixin) {
+  'use strict';
+
+  var CUTOFF_AGE = 13;
+  var View = FormView.extend({
+
+    template: Template,
+    className: 'coppa-age-input',
+
+    initialize: function (options) {
+      options = options || {};
+
+      this._formPrefill = options.formPrefill;
+    },
+
+    events: {
+      'keydown': 'submitOnEnter',
+      'keyup': 'onInput',
+      'input': 'onInput'
+    },
+
+    /**
+     * Called by the parent view to determine if the user is old enough.
+     * @method isUserOldEnough
+     * @return {Boolean} true if user is old enough, false otw.
+     */
+    isUserOldEnough: function () {
+      return this._getAge() >= CUTOFF_AGE;
+    },
+
+    onInput: function () {
+      // limit age to only 3 characters
+      var age = this.$('#age');
+      age.val(age.val().substr(0, 3));
+    },
+
+    render: function () {
+      var self = this;
+
+      return p()
+        .then(function () {
+          self.$el.html(self.template(self.getContext()));
+          self.initializePlaceholderFields();
```

Comment:
Not for this PR, but maybe we should have `floating-placeholder-mixin` call `initializePlaceholderFields` automatically in an `afterVisible` function? Dunno if that would simplify the logic in views that use it.